packages/palette_generator/lib/palette_generator.dart

```diff
@@ -1106,8 +1106,56 @@ class _ColorCutQuantizer {
 
     final ByteData imageData =
         await image.toByteData(format: ui.ImageByteFormat.rawRgba);
+    if(filters==null)
+      return await compute(_quantizefromByte, {
+        "byteData":imageData,
```

Comment:
String literals should all be single-quote strings.

Reply:
Done, thanks